### ISSUE TITLE
NullPointerException when empty visibility params in xsd

### ISSUE THREAD

**Issue by tutotio** [alkacon/opencms-core#443]
When you configure visibility params in xsd with empty list to get an element never displayed: 
http://documentation.opencms.org/opencms-documentation/content-in-opencms/defining-a-content-type/defining-the-contents-behavior/adjusting-the-contents-appearance-in-the-content-editor/hiding-schema-elements/

```
<visibilities>
    <visibility element="GalleryTitle" params="" />
</visibilities>
```

Gets a NullPointerException in some cases when you save content.

To get this error do this in OpenCms (10.0.1 or 10.5Beta) Apollo Demo:
1 - Login with Admin user.
2 - Change configuration to empty params in visibility for section in section.xsd schema to GalleryTitle:
`<visibility element="GalleryTitle" params="" />`
3 - Add new Content Section. Type Headline, Text and add an image.
4 - Save and Exit content 

Get NullPointerException

**Comment by aKandzior** [comment]
We will check and fix this for 10.5.

**Comment by dSeidel** [comment]
The issue is fixed in OpenCms 10.5. Thanks for the hint.
